ocp-indent.opam

```
@@ -37,7 +37,7 @@ depends: [
   "base-bytes"
 ]
 post-messages: [
-  "This package requires additional configuration for use in editors. Install package 'user-setup', or manually:
+  "If you're not using ocaml-lsp-server and ocamlformat, this package requires additional configuration for use in editors. Install package 'user-setup', or manually:
```

This comment is also confusing in its own way, as it implies that if you use `ocaml-lsp-server` (and `ocamlformat`) there is nothing to do to use this tool, which is not true. I would go one step further:
```suggestion
  "This tool is an alternative to `ocamlformat`. It requires additional configuration for use in editors. Install package 'user-setup', or manually:
```

---

Good suggestion - thanks @nojb.